lists/gracedShoppingList.Md

```diff
@@ -1,5 +1,10 @@
-cheese
-bread
-orange juice
-coffee
+cheese 
+bread 
+orange juice 
+coffee 
 chocolate
+milk  
+liver 
```

Comment:
@gdangelo8 are you sure you need liver? I thought you were vegetarian!

Reply:
Thank you for the feedback, I corrected this point at this location: https://github.com/pbuttigieg/shoppingList/pull/25/commits/cd060ccff0feaecf0fcd5eb68aa3d45e8e9b2f76